mmcv/utils/pytorch_wrapper.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+TORCH_VERSION = torch.__version__
+
+
+def is_cuda() -> bool:
+    return torch.cuda.is_available()
+
+
+def is_mlu() -> bool:
```

Comment:
Should we rename `is_mul` to `is_mlu_pytorch`

Reply:
It is more accurate, but it seems to be inconsistent with `is_cuda`, so I don't think it should be renamed.